Dockerfile

```diff
@@ -67,15 +67,15 @@ COPY requirements.txt ./
 RUN pip install --no-cache-dir -r requirements.txt \ 
   && apk del build-deps
 
+# Copy frontend assets.
```

Comment:
On my system these `COPY` commands execute almost instantly (maybe 100ms or so). Does it make more difference on yours?

Reply:
It's pretty quick on my system too < 1s.